modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data with rule
+You can configure the `FlowCollector` to filter eBPF flows using a global rule as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
```

Comment:
So here we are assuming a flowcollector already exists with name `cluster` Is that what is expected?

Reply:
Hrrrm. Ok, so yea the *cluster* bit here is coming from the fact that when I'm looking at this in the Web Console, I see *cluster* , so is that maybe what its named in all the pre-configured settings @Amoghrd ?

Reply:
Okay I see its the same way for all other features too. So can ignore this